platform/api.search/src/org/netbeans/modules/search/BasicSearchForm.java

```diff
@@ -877,7 +876,6 @@ public void addSeparator() {
             c.insets = new Insets(5, 5, 5, 5);
             c.fill = GridBagConstraints.HORIZONTAL;
             JSeparator separator = new JSeparator(JSeparator.HORIZONTAL);
-            separator.setForeground(SystemColor.controlShadow);
```

Comment:
Not sure which separator this is. Does removing this make sense for other LAFs as well?

Reply:
This are the two horizontal separators in the "Find in Projects" dialog (see screenshots above). In Windows LAF there is no difference because default separator foreground is already controlShadow. Did not check other LAFs, but even if there is difference, this should be no problem. Its just a simple separator in a dialog. Default separator colors should work fine in dialogs.

Reply:
Makes sense.